Dockerfile

```
@@ -1,6 +1,6 @@
 # Build image
 # Necessary dependencies to build Parrot
-FROM rust:slim-bullseye as build
+FROM rust:1.65-slim-bullseye as build
```

```suggestion
FROM rust:slim-bullseye as build
```
Let's keep this as it is. This is kept like this to "force" us to be on rust's latest version.

---

Sure, but then what's the difference between this and the Build workflow? Should we force the build workflow to use the latest version as well?

---

That's exactly why we also have the following:
```yaml
     matrix:
        rust-version: ['1.65', 'stable']
```

---

Sorry, I'm not too familiar with the actions syntax. Does this mean it tries to use 1.65 then stable? Why not have stable only?

---

AFAIK we we're using the minimal version we support plus the latest stable one. Therefore, Parrot should work from 1.65 onwards to stable (or at least in 1.65 and stable, with 1.65 being the minimal, nothing below that is supported)